frontend/justfile

```diff
@@ -25,6 +25,23 @@ export FRONTEND_PNPM_VERSION := `grep 'packageManager' ../package.json | awk -F'
 @pnpm-version:
     echo $FRONTEND_PNPM_VERSION
 
+##########
+# Docker #
+##########
+
+# This section exists because the frontend is not a part of the Docker-based setup
+# and cannot be built easily like `just dc build <service>`.
+
+# Build the frontend Docker image
+build-img tag="openverse-frontend:local" target="app":
+    docker buildx build \
```

Comment:
Can you add a note in the frontend documentation about needing to install (and potentially enable) `docker buildx`? [It's included by default in Docker Desktop but requires a separate step for Docker Engine](https://docs.docker.com/build/architecture/#install-buildx):

> Docker Engine version 23.0 and later requires that you install Buildx from a separate package

The error message you get if you do _not_ install buildx is nasty and unhelpful too:

```
➜  openverse git:(frontend_no_mv) just frontend/build-img
docker buildx build --target app --build-context 'repo_root=..' --build-arg FRONTEND_NODE_VERSION=$(just node-version) --build-arg FRONTEND_PNPM_VERSION=$(just pnpm-version) --tag openverse-frontend:local .
unknown flag: --target
See 'docker --help'.
```

Nothing to indicate (directly) that you don't have buildx installed.

Reply:
@sarayourfriend it's installed as a separate package, yes, but it's included in all the [installation methods](https://docs.docker.com/engine/install/). If you're on Docker Engine v23 and above, and followed the instructions verbatim, you have Buildx and `docker build` is an alias for `docker buildx build`.

I've added a note about needing Buildx to the docs in bfb6f83.